packages/@aws-cdk/aws-ecs/lib/linux-parameters.ts

```diff
@@ -67,6 +91,8 @@ export class LinuxParameters extends CoreConstruct {
 
     this.sharedMemorySize = props.sharedMemorySize;
     this.initProcessEnabled = props.initProcessEnabled;
+    this.maxSwap = props.maxSwap;
```

Comment:
We should validate these values. Validation logic can either be in this constructor, or in the TaskDefinition constructor when the LinuxParameters is actually used.

Reply:
I added validation with positive integer values for maxSwap and values between 0 and 100 for swappiness. I also added a test to expect the constructor to throw with out of range values.